documentation/site/pages/provers/quick-start.mdx

```diff
@@ -31,101 +35,157 @@ For a quick set up of Boundless dependencies on Ubuntu 22.04 LTS (see [Operating
 ```
 
-### Using the Bento CLI
+### Start Broker
 
-Once everything is up, you can send a test proof to Bento to see if everything works. There are two options for test proofs:
 
-1. The Bento CLI allows for a simple test vector with a specified cycle count via the `-c` flag:
+You can now start broker (which runs both bento + broker i.e. the full proving stack!):
 
 ```sh [Terminal]
-RUST_LOG=info bento_cli -c 32
+just broker
 ```
 
-2. The Bento CLI also accepts [existing program and input files](/developers/tutorials/build):
+To check the proving logs, you can use:
 
 ```sh [Terminal]
-RUST_LOG=info bento_cli -f ./crates/bento-client/method_name -i /tmp/input.bin
+just broker logs
 ```
+### Stop Broker
 
-## Depositing Funds to the Market
+To stop broker, you can run:
 
-:::note[Note]
-* To apply for access to HP tokens and become a prover, please fill out this [form](https://docs.google.com/forms/d/e/1FAIpQLScr5B3TZfzLKIb0Hk6oqiMMXdRh4cwpTlczi_zGqdwabvbrfw/viewform) with your details.
-* To read more about depositing funds to the market, and to learn about the hitpoints token, please see [Hitpoints (HP) Tokens](/provers/broker#hitpoints-hp-tokens).
-:::
+```sh [Terminal]
+just broker down
+```
+
+Or remove all volumes and data from the service:
+
+```sh [Terminal]
+just broker clean
+```
+
+## Configuration
 
-Once the proving stack is running, the next step is to deposit funds to the Boundless market contract. This is for the Broker to be able to cover staking during lock-in. Currently, this will require depositing special non-transferable hitpoints (HP) tokens which are sent to whitelisted provers.
+### Custom Environment
 
-To deposit to the market, please follow the [Deposit Stake to Market](/provers/broker#deposit--balance) instructions.
+To run broker with a custom environment file:
+
+```sh [Terminal]
+just broker up ./.env.broker
+just broker down ./.env.broker
+```
+
+### Compose.yml
+
+The `compose.yml` file defines all services within Bento. The Boundless repo includes a starter `compose.yml` which you can see [here](https://github.com/boundless-xyz/boundless/blob/main/compose.yml). For Broker configuration options, please see [Broker.toml](/provers/broker#settings-in-brokertoml).
+
+### Multi GPU
+
+In `compose.yml`, you can configure the number of GPU's via adding new `gpu_agent<I>` and changing the `device_ids` to map to specific physical GPUs (if on a single host, multi-GPU config).
+
+### Multi Host
+
+Services can be run on other hosts, as long as the IP addresses for things link PostgreSQL / Redis / MinIO are updated on the remote host. See the `.env.broker-template` HOST configuration options [here](https://github.com/boundless-xyz/boundless/blob/main/.env.broker-template) to adjust them.
+
+### Segment Size
+
+Under the `exec_agent` service you can configure the segment size with the `--segment-po2` flag. Bigger segments are preferable for performance but do impact the proving systems conjectured security bits slightly.
```

Comment:
maybe best indicated that you would configure this with `SEGMENT_SIZE` env variable when starting bento/broker. This defaults to 21 and passes the value into this flag in the compose. This might be confusing that users would expect to add this flag to the `just` commands